### PR TITLE
[dist test feat] Added v2 change API. Bump deps

### PR DESCRIPTION
**Pull request by fritzmonkey**
https://github.com/warehouseai/feedsme-api-client/pull/3 and https://github.com/warehouseai/carpenterd-api-client/pull/2 should be merged first, so the API clients can be bumped here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +72,7 @@ a 2 different scenarios that influence our course of action.
 
 Below we have diagrams of specific cases that we handle.
 
-## Dependent Duilds Diagram
+## Dependent Builds Diagram
```
Comment (msluther) [inline README.md:75]: You didn't like the portmanteau of "Dependent & Builds"? 😂

File: lib/feedsme/index.js [hunk 2]
```diff
@@ -28,10 +28,17 @@ function Feedsme({ agents, models, config, log }) {
   var proto = url.parse(uri).protocol;
 
   this.wrhs = new Warehouse(config.get('warehouse'));
-  this.carpenter = new Carpenter({
-    uri: uri,
-    agent: agents[proto]
-  });
+  this.carpenter = {
+    v1: new Carpenter({
```
Comment (msluther) [inline lib/feedsme/index.js]: Does feedsme still need a v1 client?

Comment (msluther) [reply lib/feedsme/index.js]: shouldn't this just be plumbing the `promote` bit through everything? And the feedsme v1 endpoint just always sets `promote=true` but always still talks to the v2 carpetnerd endpoint?

File: package.json [hunk 3]
```diff
@@ -8,11 +8,12 @@
   },
   "scripts": {
     "test": "nyc --reporter=text --reporter=lcov mocha test/*.test.js",
```
Comment (msluther) [inline package.json]: make this be
```
"test": "nyc --reporter=text --reporter=lcov npm run test:mocha",
```

File: package.json [hunk 4]
```diff
@@ -8,11 +8,12 @@
...
+    "test:mocha": "mocha test/*.test.js",
     "watch": "mocha --watch test/*.test.js",
```
Comment (msluther) [inline package.json]: make this be:
```
"watch": "npm run test:mocha -- --watch",
```

**Review by msluther — CHANGES_REQUESTED** [review]

File: test/feedsme.test.js [hunk 5]
```diff
@@ -178,6 +181,89 @@ describe('feedsme', function () {
...
+
+      it('accepts a promotion option', function (done) {
+        sinon.stub(app.feedsme, 'change');
+
+        const data = { name: 'pkg' };
+
+        request.post({
+          uri: url.resolve(root, '/v2/change/dev'),
+          json: true,
+          body: { promote: false, data }
```
Comment (msluther) [inline test/feedsme.test.js:255]: tests for `body: { data }` and `body: { promote: true, data }` both promote the build?

**Review by msluther — APPROVED** [review]
LGTM other than a suggested test and some npm script cleanup that can happen.

File: README.md (hunk 1 above)
Comment (fritzmonkey) [reply README.md:75]: I assumed it was a typo. 😆

**Review by msluther — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jcrugzz) [reply README.md:75]: 🤣

**Review by jcrugzz — APPROVED** [review]

**Review by jcrugzz — COMMENTED** [review]
Travis should pass and we can merge when thats true 👍

**Pull request merged**